storage/localstore/schema.go

```diff
@@ -8,7 +24,7 @@ import (
 
 // The DB schema we want to use. The actual/current DB schema might differ
 // until migrations are run.
-const CurrentDbSchema = DbSchemaSanctuary
+var DbSchemaCurrent = DbSchemaDiwali
```

Comment:
I think that it would be more in compliance with naming to have a capitalised `B` in `Db` in all `DbSchema` variable names.

Reply:
compliance with which other convention? you originally added it with `Db` not `DB` :)

Reply:
as an example we also usually use `Uid` not `UID`, `Ruid` as opposed to `RUID`, etc. if anything, we should first conclude on how we capitalize acronyms in the first place

Reply:
I may be wrong, but I do not think that I added schema name variables. I used DB in shed. 

The convention that is usually noted is this one https://github.com/golang/go/wiki/CodeReviewComments#initialisms on initialisms. I also think that `Uid` should be `UID` and so on, but I missed to review these pull requests.

The uppercaps argument does not stand, as it is about "same" caps, `DB` can be `db` at the start of unexported value or on its own.

In any case, we have so much inconsistencies in the code that this does not matter at all.

Reply:
cool, i agree with you, let's stick to the golang reccomendations for next PRs. we can actually do this change as a separate PR and rename across the codebase for consistency